core/templates/filters/limit-to.pipe.ts

```diff
@@ -13,27 +13,16 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive for warning and alerts.
+ * @fileoverview limitTo filter for Oppia.
```

Comment:
Instead of just writing limitTo filter, maybe write what this does. (A pipe to limit the number elements in an array or something like that)

Reply:
done